### PR TITLE
#1956 Make project table look like change request table

### LINKED ISSUE #1956: Make Project Table Look Like Change Request Table
### Description

We've updated the change request table to look a little better and be a little cleaner. We want to update the projects table to be the same. This ticket is to make the projects ticket look like the change request table.

### Acceptance Criteria

- [x] Project Table Looks like the Change Request Table
- [x] All Other functionality is unaffected

### Proposed Solution

This ticket is going to be involved but you should be able to get a lot from looking at the change request table code.

### Mocks

Current Projects Table: 
image
How we want it to look:
image

### PR DESCRIPTION
**Pull request by alaneng-neu**
## Changes

Made project table look like CR table

## Screenshots

image
image

## Checklist

It can be helpful to check the `Checks` and `Files changed` tabs.
Please review the [contributor guide](https://nerdocs.atlassian.net/wiki/spaces/NER/pages/8060929/Software+Contributor+Guide) and reach out to your Tech Lead if anything is unclear.
Please request reviewers and ping on slack only after you've gone through this whole checklist.

- [x] All commits are tagged with the ticket number
- [x] No linting errors / newline at end of file warnings
- [x] All code follows repository-configured prettier formatting
- [x] No merge conflicts
- [x] All checks passing
- [x] Screenshots of UI changes (see Screenshots section)
- [x] Remove any non-applicable sections of this template
- [x] Assign the PR to yourself
- [x] No `yarn.lock` changes (unless dependencies have changed)
- [x] Request reviewers & ping on Slack
- [x] PR is linked to the ticket (fill in the closes line below)

Closes #1956

### CODE REVIEW

File: src/frontend/src/pages/ProjectsPage/ProjectsTable.tsx [hunk 1]
```diff
@@ -104,14 +125,7 @@ const ProjectsTable: React.FC = () => {
...
-      field: 'status',
-      headerName: 'Status',
-      type: 'singleSelect',
-      valueOptions: Object.values(WbsElementStatus),
-      maxWidth: 100
-    }
+    statusColumn
   ];
 
   const filterValues = JSON.parse(
```
Comment (RChandler234) [inline src/frontend/src/pages/ProjectsPage/ProjectsTable.tsx:131]: I'm getting an undefined error on this line:
image

**Review by RChandler234 — CHANGES_REQUESTED** [review]

File: src/frontend/src/pages/ProjectsPage/ProjectsTable.tsx (hunk 1 above)
Comment (alaneng-neu) [reply src/frontend/src/pages/ProjectsPage/ProjectsTable.tsx:131]: huh thats weird because it looks like valid json and also it runs fine on my machine

Comment (RChandler234) [reply src/frontend/src/pages/ProjectsPage/ProjectsTable.tsx:132]: note for self later: 
image

**Review by RChandler234 — CHANGES_REQUESTED** [review]
my projectsTableFilter in localstorage is getting set to the string 'undefined', it looks like the problem is on line 225

File: src/frontend/src/pages/ProjectsPage/ProjectsTable.tsx [hunk 2]
```diff
@@ -168,7 +222,7 @@ const ProjectsTable: React.FC = () => {
           }
         }}
         onFilterModelChange={(filterModel: GridFilterModel) => {
-          localStorage.setItem('projectsTableFilter', JSON.stringify(filterModel.items[0]));
+          localStorage.setItem('projectsTableFilter', JSON.stringify(filterModel.items[0] ?? null));
```
Comment (RChandler234) [inline src/frontend/src/pages/ProjectsPage/ProjectsTable.tsx:225]: getting the same problem but for null, I think we'll need to do a ternary and not set the localStorage if the filterModal.items[0] is undefined or null:
image

**Review by RChandler234 — CHANGES_REQUESTED** [review]

**Review by RChandler234 — APPROVED** [review]
LGTM 🔥

**Pull request merged**